examples/nlp/language_modeling/megatron_gpt_eval.py

```diff
@@ -311,17 +335,17 @@ def main(cfg) -> None:
         "end_strings": cfg.inference.end_strings,
     }
 
+    prompts = load_prompts(cfg)
+
     fp8_enabled = hasattr(model.cfg, "fp8") and (model.cfg.fp8 == True)
-    if fp8_enabled:
-        nb_paddings = 0
-        while len(cfg.prompts) % 8 != 0:
-            cfg.prompts.append("")
-            nb_paddings += 1
+    if fp8_enabled and len(prompts) > 0:
+        padded_len = ((len(prompts) + 7) // 8) * 8
```

Comment:
I wonder if it possible to avoid that hardcoded numbers?

Reply:
sure let me make this a function